dist/index.html

```diff
@@ -9,8 +9,7 @@
     <nav class="nav-bar">
         <img class="logo" id="logo" src="./images/turing-logo.png" alt="Turing Logo placeholder">
         <menu class="nav-links">
-            <li id="nav-search">Search Recipes</li>
-            <input type="text" placeholder="Search.." id="searchBar">
+            <input type="text" placeholder="Search Recipes.." id="searchBar">
```

Comment:
Still want to add a button to process the search here!

Reply:
yea looks good!